bin/next-start

```diff
@@ -18,10 +19,8 @@ const argv = parseArgs(process.argv.slice(2), {
 const dir = resolve(argv._[0] || '.')
 
 const srv = new Server({ dir })
-srv.start(argv.port)
-.then(() => {
-  console.log('> Ready on http://localhost:%d', argv.port)
-})
+
+run(srv, argv.port)
```

Comment:
I agree, let's just throw an error in `next-start`, will update. Throwing an error is still better than existing behavior, currently it will throw nothing and the previous owner of the port will simply be the only one bound.